scss/mixins/_viewport-logic.scss

```diff
@@ -0,0 +1,176 @@
+@import "susy";
+@import "maptovars";
+
+$container-size: $current-columns * $column-width;
+
+$susy: (
+  container: $container-size,
+  columns: $current-columns,
+  gutters: 20px/80px,
+  column-width: $column-width,
+  wrapper: 1400px,
+  global-box-sizing: content-box,
+  gutter-position: inside-static
+);
+
+ .wrapper_#{$columns-large} {
+   max-width: 1400px;
+ }
+
+.container_#{$columns-large} {
+  @include container();
+
+  &.breakout-container {
+    max-width: 100%;
+
+    .grid-zone.grid_#{$columns-large} {
+      margin-left: calc((100% - #{$container-size}) / 2);
+      margin-right: calc((100% - #{$container-size}) / 2);
+      width: $container-size;
+    }
+  }
+}
+
+$zone-sizes: $grid-zones;
+@each $zone-index in $zone-sizes {
+  .grid-zone.grid_#{$zone-index} {
+    @if $zone-index <= $current-columns {
```

Comment:
Als deze `if` `false` is levert dit in de css een lege rule op. Je zult de `if` OM de hele rule willen zetten denk ik?